decidim-participatory_processes/app/cells/decidim/participatory_processes/process_filters_cell.rb

```diff
@@ -100,7 +100,7 @@ def explanation
         content_tag(
           :span,
           I18n.t(explanation_text, scope: "decidim.participatory_processes.participatory_processes.filters.explanations"),
-          class: "muted mr-s ml-s"
+          class: "mr-s"
```

Comment:
We can drop `mr-s` too (it's on the `erb-lint.yml`)

```suggestion
```

Reply:
addressed in https://github.com/decidim/decidim/pull/12138/commits/cbbb3fb6628355991a36462225c5b0ab9f8adf67